lib/oneaws/cli.rb

```diff
@@ -51,5 +51,21 @@ def getkey
         EOS
       end
     end
+
+    private
+
+    # AWS の credential を以下の順番で存在チェックをする
+    # 1. ~/.aws/credentials
+    # 2. ~/.aws/config/credentials
+    # 存在しない場合は順番1つ目のものを用いる
+    def find_credentials
+      credentials = ["~/.aws/credentials", "~/config/aws/credentials"]
```

Comment:
コメントを見ると `.config` でしょうか？
```suggestion
      credentials = ["~/.aws/credentials", "~/.config/aws/credentials"]
```